staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -270,7 +272,7 @@ var (
 			Buckets:        requestDurationSecondsBuckets,
 			StabilityLevel: compbasemetrics.ALPHA,
 		},
-		[]string{priorityLevel, flowSchema},
+		[]string{priorityLevel, flowSchema, "watch"},
```

Comment:
@logicalhan - I'm assuming we can still modify alpha-stability metrics?

Reply:
This should be fine, but we should add a release note about it.